test/remote/account_create_tests.js

```diff
@@ -337,20 +331,15 @@ describe('remote account create', function() {
         redirectTo: 'http://accounts.firefox.com.evil.us'
       }
       return api.accountCreate(email, authPW, options)
-      .then(
-        assert.fail,
-        function (err) {
+      .then(() => { assert.fail }, (err) => {
```

Comment:
This should be `.then(assert.fail, (err) => {`. Doing it this way will correctly fail the test if the request succeeds.